docs/backend-api/how-to/how to get track points.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Get session hash
+description: About session hash key
+---
+
+# How to get track points for trips
+
+Sometimes necessary to get all points of a trip with more info about the device's moves.
+How to get them?
+
+Firstly you need to [get hash](../get-session-hash.md).
+
+Once you get the hash, you need to [get your tracker_id](../get-tracker-list/). The platform must know points for what device must be in reply.
+
+Now you can get all points for the interesting period using track/read API call:
+
+    [api_base_url]/track/read?hash=your_hash&tracker_id=device's_id&from=yyyy-MM-dd HH:mm:ss&to=yyyy-MM-dd HH:mm:ss
+
+Parameters that necessary for this call:
+
+* tracker_id - we got them in [tracker/list](../tracker.md#list) call. Use only one tracker_id per call. It should be an integer.

```

Comment:
The relative link from this place will be: `../resources/tracking/track/track.md#list`.